L81-aspect-based-python-bazel-rules.md

```diff
@@ -0,0 +1,50 @@
+Aspect-based Python Bazel Rules
+----
+* Author(s): Michael Beardsworth
+* Approver: a11r
+* Status: Draft
+* Implemented in: Bazel and Starlark
+* Last updated: [Date]
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Don't forget to create the thread and link it on this line.

Reply:
Ack. Should I do that in a follow-up before landing the PR?

Reply:
Yes. The merged PR of the gRFC should include a link to the email thread. The process is:

1. Create the PR with the link not included. (like you have here)
2. Create the discussion at `grpc-io` with the link to the PR just created.
3. Update the PR to include the link to the discussion.
4. Wait 2 weeks.
5. Merge PR.

Reply:
Gotcha, thanks.